reana_server/rest/config.py

```diff
@@ -0,0 +1,72 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of REANA.
+# Copyright (C) 2020 CERN.
+#
+# REANA is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Reana-Server config-functionality Flask-Blueprint."""
+
+import logging
+import traceback
+
+from flask import Blueprint, jsonify
+from reana_commons.config import REANAConfig
+
+
+blueprint = Blueprint("config", __name__)
+
+
+@blueprint.route("/config", methods=["GET"])
+def get_config():
```

Comment:
Right now we have what seems a generic endpoint which returns REANA-UI only config. Therefore, it can be a bit misleading, I see two solutions:
1. We return a more complex object, where each key is the type of config, and inside there is the specific config payload. Example (I am just making up some config to illustrate):
```json
{
  "ui": {
    "local_users": true,
    ...
  },
  "cli": {
    "colored_output": "true"
    ...
  }
}
```
2. Take a parameter in the endpoint which determines which kind of config should be retrieved.